spreadsheet_dashboard_account_account_oca/static/tests/tours/accounting_dashboard_tours.esm.js

```diff
@@ -0,0 +1,6 @@
+/** @odoo-module */
+
+import {registerDashboardTour} from "@spreadsheet_dashboard/../tests/utils/tour";
+
+registerDashboardTour("spreadsheet_dashboard_accounting_oca", "Accounting");
```

Comment:
The strategy to test dashboards changed from tours to a more comprehensive python test [1]
You don't have anything to do. All dashboards are validated post install :wink: [2]

[1] https://github.com/odoo/odoo/commit/d72fa35cedb1a2b9e6b589a4d6e5144945ea4e81
[2] https://github.com/odoo/odoo/commit/d72fa35cedb1a2b9e6b589a4d6e5144945ea4e81#diff-8f2fab37a40db5a41686130eed8834dd64005fc6bd03234a3f17c9ea38bf887eR11